cli/assets/templates/porth/Makefile

```diff
@@ -0,0 +1,55 @@
+ifndef WABT_PATH
+$(error Download Wabt (https://github.com/WebAssembly/wabt) and set $$WABT_PATH)
+endif
+
+4ORTH = 4orth
+4ORTH_PATH  = $(shell command -v  $(4ORTH))
+
+ifeq (, $(4ORTH_PATH))
+$(error Download 4orth (https://github.com/FrankWPA/4orth) and add it to $$PATH)
+endif
+
+4ORTH_ARGS = com -s
+
+ifndef _4ORTH
+4ORTH_STD = $(shell dirname -- "$(4ORTH_PATH)")/std
+4ORTH_ARGS := -I $(4ORTH_STD) $(4ORTH_ARGS)
+endif
+
+ifeq ($(b), 1)
+4ORTH_ARGS := $(4ORTH_ARGS) -b
+endif
+
+ifeq ($(r), 1)
+4ORTH_ARGS := $(4ORTH_ARGS) -r
+endif 
```

Comment:
What do these options do? I wonder if we should have better names for the environment vars?

Reply:
Oh, I did use those to be the same as 4orth compile [options](https://github.com/FrankWPA/4orth#running-options-and-subcommands). We could change the names or put a comment to use `4orth help` to see the command descriptions.
Or we can add aliases as `run` and `bundle`. I'd like to keep the short ones too as it's quicker to type rapidly for a test.